packages/collections/README.md

```diff
@@ -0,0 +1,38 @@
+# Collections
+
+[![Travis CI Status](https://img.shields.io/travis/monumentjs/collections/master.svg?logo=travis)](https://travis-ci.org/monumentjs/collections)
```

Comment:
Travis runs on `core` repository; please fix links (there should be `core` instead of `collections`)

Reply:
Fixed👌